kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/NavigationService.java

```diff
@@ -195,9 +196,9 @@ public String getRedirectCodeByUser() {
 		String defCode = "";
 		try {
 			BaseEntity user = beUtils.getUserBaseEntity();
-			List<EntityAttribute> priIsAttributes = beaUtils.getBaseEntityAttributesForBaseEntityWithAttributeCodePrefix(user.getRealm(), user.getCode(), PRI_IS_PREFIX);
+			Set<EntityAttribute> priIsAttributes = beaUtils.getBaseEntityAttributesForBaseEntityWithAttributeCodePrefix(user.getRealm(), user.getCode(), PRI_IS_PREFIX);
 			if (priIsAttributes.size() > 0) {
-				EntityAttribute attr = priIsAttributes.get(0);
+				EntityAttribute attr = priIsAttributes.iterator().next();
```

Comment:
Can we move away from PRI_IS please?

Reply:
Should be using LNK_DEF

Reply:
To be addressed as part of a dedicated Jira ticket.